MUXSDKStats/MUXSDKStats/MUXSDKStats+Internal.h

```diff
@@ -0,0 +1,23 @@
+//
+//  MUXSDKStats+PropertyGetters.h
+//  MUXSDKStats
+//
+//  Declares a project-level class extension that allows tests to get values under test out of the SDK
+//
+//  Created by Emily Dixon on 10/17/22.
+//  Copyright © 2022 Mux, Inc. All rights reserved.
+//
+
+#ifndef MUXSDKStats_PropertyGetters_h
+#define MUXSDKStats_PropertyGetters_h
+
+#import "MUXSDKStats.h"
+
+@interface MUXSDKStats()
```

Comment:
Alright so what I remember about categories is that we used to use them but they don't get linked up correctly when building the static framework. It ends up crashing at runtime. 

I can't remember all access to the class is affected or just if the method on the category is called - perhas worth testing? And also if we do use this, can we please add a warning comment to this category warning of the pitfalls of using it outside of tests? It makes me a bit nervous to include it